app/views/people/show.html.haml

```diff
@@ -21,29 +21,35 @@
             %p= link_to("#{t('all').capitalize} #{t('current')} #{t('mp_plural')}", house_members_current_a_z_letter_path(HousesHelper.commons_id, 'a'))
         - else
           %p
-            - if @person.statuses[:house_membership_status].include?('Lord')
+            - if @person.statuses[:house_membership_status].include?('Member of the House of Lords')
```

Comment:
There's another change (website-1248) that removes all this logic which would make this change redundant, so we are going to leave it for now